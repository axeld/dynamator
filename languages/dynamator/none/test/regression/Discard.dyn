<dynamator language="none">
  <!-- Test discard and discard-tag -->
  <class name="discard_x">
    <discard/>
  </class>
  <class name="discard_tag">
    <discard-tag/>
  </class>
  <id name="test">
    <before>
      Shouldn't see this! (before)
    </before>
    <raw-content>
      Shouldn't see this! (raw-content)
    </raw-content>
  </id>
</dynamator>
