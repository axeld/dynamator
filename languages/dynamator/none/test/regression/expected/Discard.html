<html>
  <head>
    <title>Test Discard</title>
  </head>
  <body>
  </body>
</html>
<!-- generated by Dynamator Sun Feb 15 17:23:23 CST 2004 -->
